docs/index.js

```diff
@@ -47,12 +47,12 @@ const renderProps = (props: Object): Array<React.Element<any>> => {
     const prop = props[key]
     return (
       <Item key={index}>
-        <Text small blue bold>{key}</Text>
+        <Text size={'small'} colour={'blue'} bold>{key}</Text>
```

Comment:
This doesn't need to be surrounded by curly braces, your just passing a string so we can just pass it in as a string:
```javascript
<Text size='small' colour='blue' bold>{key}</Text>
```